code/modules/mob/living/carbon/human/unarmed_attack.dm

```diff
@@ -14,7 +14,8 @@ var/global/list/sparring_attack_cache = list()
 	var/shredding = 0 // Calls the old attack_alien() behavior on objects/mobs when on harm intent.
 	var/sharp = 0
 	var/edge = 0
-	var/delay = 1 SECOND	//Default delay, overrideable
+	var/def_delay = 1 SECOND //Default delay
```

Comment:
scrap these extra vars, use the attackspeed statmod to handle this, it should already work with unarmed attacks